Mathlib/CategoryTheory/Limits/Preserves/Finite.lean

```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Andrew Yang
 -/
 import Mathlib.CategoryTheory.Limits.Preserves.Basic
-import Mathlib.CategoryTheory.FinCategory.AsType
+import Mathlib.CategoryTheory.Limits.Shapes.FiniteProducts
```

Comment:
Without this import change, this file doesn't know about the classes `HasFiniteLimits` and `HasFiniteProducts` etc.